x-pack/plugins/security_solution/server/config.test.ts

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { configSchema } from './config';
+
+describe('config', () => {
+  describe('alertMergeStrategy', () => {
```

Comment:
```suggestion
  describe('alertIgnoreFields', () => {
```

Reply:
Thanks for pointing these out. I will fix 'em here.